docs/dev-guide/validation.md

```
@@ -0,0 +1,17 @@
+# Validation
+
+Currently, there is no automated testing in place. While automation is being
+considered for future development, the following tests should be performed
+manually:
+
+* Verify that the following features work:
+    * Changing display themes
```

Markdownlint

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)